### PR TITLE
Prettier README

### PR DESCRIPTION
**Pull request by kkweon**
- Indent images in the list
  - Indent matters in markdown.
- Ran [prettier](https://prettier.io/) partially
- Edit the image to show where to click

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,19 +3,20 @@
...
 
 1. README에 나열된 목록에서 "미착수" 상태인것 중 번역을 희망하는 글에 대하여 "착수"로 변경합니다.
-   - 웹브라우져상에서 직접적으로 Pull Request를 보내기를 희망하시는 분은 아래 스크린샷 처럼, 우측의 연필모양의 아이콘을 클릭해서 파일내용의 수정이 가능합니다.
 
-![edit](./resources/images/edit.png)
+  - 웹브라우져상에서 직접적으로 Pull Request를 보내기를 희망하시는 분은 아래 스크린샷 처럼, 우측의 연필모양의 아이콘을 클릭해서 파일내용의 수정이 가능합니다.
+    ![edit](./resources/images/edit.png)
 
-2. 1번 항목에서 README 파일의 수정이 완료되면, 아래 스크린샷 처럼 화면 하단에 변경사항을 Commit 하기위한 양식이 있습니다. 여기서, 타이틀 및 코멘트의 작성을 하고, 옵션 버튼의 두 번째를 선택하여 커밋이 새로운 브랜치에서 이뤄지도록 합니다. "Propose file change" 버튼을 클릭합니다.
+1. README 파일의 수정이 완료되면, 아래 스크린샷 처럼 화면 하단에 변경사항을 Commit 하기위한 양식이 있습니다. 여기서, 타이틀 및 코멘트의 작성을 하고, 옵션 버튼의 두 번째를 선택하여 커밋이 새로운 브랜치에서 이뤄지도록 합니다. "Propose file change" 버튼을 클릭합니다.
```
Comment (deep-diver) [inline README.md]:
```suggestion
2. README 파일의 수정이 완료되면, 아래 스크린샷 처럼 화면 하단에 변경사항을 Commit 하기위한 양식이 있습니다. 여기서, 타이틀 및 코멘트의 작성을 하고, 옵션 버튼의 두 번째를 선택하여 커밋이 새로운 브랜치에서 이뤄지도록 합니다. "Propose file change" 버튼을 클릭합니다.
```

I know 1. 1. 1. 1. is preferred way to express numbered list, but looks like there is something wrong here I guess.
currently they are displayed as 1. 1. 1. 2. which are supposed to be like 1. 2. 3. 4.

File: README.md [hunk 2]
```diff
@@ -3,19 +3,20 @@
...
 
-![commit_newbranch](./resources/images/commit_newbranch.png)
+  ![commit_newbranch](./resources/images/commit_newbranch.png)
 
-3.  Pull Request에서 deep-diver 계정 사용자를 반드시 리뷰어로 등록해 주시고, 이후 Pull Request가 수리될 때 까지 리뷰과정을 거치게 됩니다. 
+1. Pull Request에서 deep-diver 계정 사용자를 반드시 리뷰어로 등록해 주시고, 이후 Pull Request가 수리될 때 까지 리뷰과정을 거치게 됩니다.
```
Comment (deep-diver) [inline README.md]:
```suggestion
3. Pull Request에서 deep-diver 계정 사용자를 반드시 리뷰어로 등록해 주시고, 이후 Pull Request가 수리될 때 까지 리뷰과정을 거치게 됩니다.
```

same thing here.

File: README.md [hunk 3]
```diff
@@ -3,19 +3,20 @@
...
 
-4. 리뷰가 완료되면, 번역해주신 내용이 master 브랜치로 merge되어 등재됩니다. 등재된 글에 대하여 README 파일의 링크 추가 및 상태를 "착수" => "완료"로 변경에 대한 Pull Request도 보내주시기 바랍니다. (만약 이 과정을 생략하면, 일정시간이 지난 후 관리자가 일괄적으로 업데이트할 것입니다)
+1. 리뷰가 완료되면, 번역해주신 내용이 master 브랜치로 merge되어 등재됩니다. 등재된 글에 대하여 README 파일의 링크 추가 및 상태를 "착수" => "완료"로 변경에 대한 Pull Request도 보내주시기 바랍니다. (만약 이 과정을 생략하면, 일정시간이 지난 후 관리자가 일괄적으로 업데이트할 것입니다)
```
Comment (deep-diver) [inline README.md]:
```suggestion
4. 리뷰가 완료되면, 번역해주신 내용이 master 브랜치로 merge되어 등재됩니다. 등재된 글에 대하여 README 파일의 링크 추가 및 상태를 "착수" => "완료"로 변경에 대한 Pull Request도 보내주시기 바랍니다. (만약 이 과정을 생략하면, 일정시간이 지난 후 관리자가 일괄적으로 업데이트할 것입니다)
```

same thing here.

**Review by deep-diver — COMMENTED** [review]
please check

**Comment by kkweon** [comment]
Oop nvm closing

**Pull request closed without merging**